src/audiences/ui/data/reducer.js

```diff
@@ -8,6 +8,9 @@ import { unionBy } from 'lodash';
  * @returns {number} Sort result value.
  */
 const sortPosts = ( a, b ) => {
+	if ( a.error || b.error ) {
```

Comment:
This is possibly an unrelated fix, however it could be related to the bug that is happening now.

Reply:
We probably shouldn’t be adding posts that are actually errors to the list in the first place. I’d filter those out in the `getPosts()` function and maybe store or set an error in the redux store instead, then we could decide what to do with it / how to display it later

Reply:
This isn't even in relation to posts. This appears to be any link in the element. It's not specific enough and it's erroring before the list is even displayed, hence at least filtering out the errored elements.

Case in point: I believe this causes an error in the "Select Audiences" link and the variant link in the block itself.

Reply:
This sort posts function is just used when adding posts or updating posts in the redux store, it shouldn’t have any relation to the lower level components - I’m saying that errors should not be added to the list of posts in the first place. You could check the response in the getPosts function or in the reducer itself to make sure of that

Reply:
¯\_(ツ)_/¯ I will check it out but it seems to be affecting more than just the post list.

Reply:
The posts are read from the list in the store so if we have an error stored instead of a post it would appear that way. I think in this instance it can be addressed at a lower level if that makes sense.